### DISCUSSION TITLE
(REQ) Add host only raw

### DISCUSSION THREAD

**Discussion by Manssizz** [realodix/AdBlockID#722] [Ideas]
request split hostname only yang tanpa filter. penerapannya untuk android device.
thank you

**Comment by realodix** [comment]
Untuk host adserver, kebeneran [EnergizedProtection/block](https://github.com/EnergizedProtection/block) sudah membuatkannya
https://block.energized.pro/assets/sources/filter/adblock-id.txt

Namun menurut Saya untuk lebih efektif di HP Android, Anda bisa menggunakan [AdGuard for Android](https://adguard.com/en/adguard-android/overview.html). AdBlockID mengandung beragam fitur lainnya seperti anti-antiadblock dan berbagai macam teknik untuk menghilangkan iklan, yang tidak dapat dilakukan hanya mengandalkan host adserver.

**Comment by Manssizz** [comment]
noted. 
thank you

**Comment by 3xploiton3** [comment]
> request split hostname only yang tanpa filter. penerapannya untuk android device.
> thank you

bisa pakai teknologi dns DOH/DOT, saya pakai layanan nextdns, lebih ringan tanpa install app
syarat android 9 keatas

**Comment by realodix** [comment]
Saya pikir pembahasan ini cukup penting untuk salah satu sumber informasi dan diskusi. Saya akan buka kembali issue ini dengan harapan orang lain yang belum mengetahui, jadi bisa mudah untuk mengetahui.

**Comment by Manssizz** [comment]
> bisa pakai teknologi dns DOH/DOT, saya pakai layanan nextdns, lebih ringan tanpa install app
> syarat android 9 keatas

betul. emang nextdns itu enak, tanpa perlu install app tambahan (ane juga pakai untuk router). ya meskipun ada limitasi 
tapi kendala sdk ver 23 aka android 6 itu masih belum support DOH/DOT, meskipun bisa (install dns changer) itu pasti resource memory yang kepakai lumayan besar, ditambah penerapan untuk android tv box yang cuma punya ram 1gb. jadi alternatif ya masukin langsung ke host ^_^

**Comment by 3xploiton3** [comment]
> > bisa pakai teknologi dns DOH/DOT, saya pakai layanan nextdns, lebih ringan tanpa install app
> > syarat android 9 keatas
> 
> betul. emang nextdns itu enak, tanpa perlu install app tambahan (ane juga pakai untuk router). ya meskipun ada limitasi
> tapi kendala sdk ver 23 aka android 6 itu masih belum support DOH/DOT, meskipun bisa (install dns changer) itu pasti resource memory yang kepakai lumayan besar, ditambah penerapan untuk android tv box yang cuma punya ram 1gb. jadi alternatif ya masukin langsung ke host ^_^

knp ga tv box di sambungkan ke jalur ruternya?
saya dulu sebelum tahu teknologi DOH/DOT/DNScrypt juga pakai host di komputer, tp ada limitnya bikin lama loadingnya,

**Comment by Manssizz** [comment]
soalnya mau diterapin di custom rom yang ada di repo ane yang [ini](https://github.com/Manssizz/CendrawasihTV) om. mungkin kalau orang orang kaya model kita ngerti tentang doh/dot, tapi kalau untuk orang yang awam kayanya cukup ngerepotin juga

**Comment by unknown4849** [comment]
> Untuk host adserver, kebeneran [EnergizedProtection/block](https://github.com/EnergizedProtection/block) sudah membuatkannya
> https://block.energized.pro/assets/sources/filter/adblock-id.txt
> 
> Namun menurut Saya untuk lebih efektif di HP Android, Anda bisa menggunakan [AdGuard for Android](https://adguard.com/en/adguard-android/overview.html). AdBlockID mengandung beragam fitur lainnya seperti anti-antiadblock dan berbagai macam teknik untuk menghilangkan iklan, yang tidak dapat dilakukan hanya mengandalkan host adserver.

^
asset energized butuh perbaikan segera `welcome.indihome.co.id` ikut kebawa , efeknya situs apapun yg direct pertama ke welcome indihome langsung mati (ke blokir di awal) . 

===============================
Disini sini pd pake NextDNS jg mantab :3 , kalo ane preff apk adguard + Dns Filter (built in- adguard) *ada perbedaan DNS filter dan Filter biasa , dns filter mirip blokir pake file host jd ga ngotorin query DNS , kalo filter biasa ga nampilin "kotak kosong" tapi ngotorin query dns. 

btw kalo mobile device kadang ada merek tertentu kurang support sama DoT (Built-in andorid) ya contoh dlu ane pernah pake somay note 9 pru coba aktifin DoT , efeknya beberapa app bawaan somay mati , berbeda setelah ganti hp pake DoT work fine cuma ga bisa ngilangin iklan kudu rooted phone.

**Comment by realodix** [comment]
> asset energized butuh perbaikan segera `welcome.indihome.co.id` ikut kebawa , efeknya situs apapun yg direct pertama ke welcome indihome langsung mati (ke blokir di awal) .

@unknown4849 Saya rasa ini perlu perbaikan di sisi Energized dengan menambahkannya ke whitelist. Di AdBlockID `welcome.indihome.co.id` ditandai dengan `$third-party`, ini membuatnya tidak menimbulkan efek yang sama.

**Comment by 3xploiton3** [comment]
> > Untuk host adserver, kebeneran [EnergizedProtection/block](https://github.com/EnergizedProtection/block) sudah membuatkannya
> > https://block.energized.pro/assets/sources/filter/adblock-id.txt
> > Namun menurut Saya untuk lebih efektif di HP Android, Anda bisa menggunakan [AdGuard for Android](https://adguard.com/en/adguard-android/overview.html). AdBlockID mengandung beragam fitur lainnya seperti anti-antiadblock dan berbagai macam teknik untuk menghilangkan iklan, yang tidak dapat dilakukan hanya mengandalkan host adserver.
> 
> ^
> asset energized butuh perbaikan segera `welcome.indihome.co.id` ikut kebawa , efeknya situs apapun yg direct pertama ke welcome indihome langsung mati (ke blokir di awal) .
> 
> ===============================
> Disini sini pd pake NextDNS jg mantab :3 , kalo ane preff apk adguard + Dns Filter (built in- adguard) *ada perbedaan DNS filter dan Filter biasa , dns filter mirip blokir pake file host jd ga ngotorin query DNS , kalo filter biasa ga nampilin "kotak kosong" tapi ngotorin query dns.
> 
> btw kalo mobile device kadang ada merek tertentu kurang support sama DoT (Built-in andorid) ya contoh dlu ane pernah pake somay note 9 pru coba aktifin DoT , efeknya beberapa app bawaan somay mati , berbeda setelah ganti hp pake DoT work fine cuma ga bisa ngilangin iklan kudu rooted phone.

klo iklan berbasis dns semua bisa dihilangkan, yg paling sering dari dns graph.facebook.com, tapi akan merusak fungsi fb app & turunannya,

saya pakai dot di pie semua gak ada masalah kecuali login ke wifi.id harus dimatikan dulu dotnya
